### ISSUE TITLE
Show bookmarks on home screen

### ISSUE THREAD

**Issue by j9t** [brave/brave-browser#26383]
## Description

TL;DR: Please show bookmarks on the Brave home screen (or enable other configurable ways to use the space).

The home screen is a great place to browse or search the Web. Showing frequently visited sites is helpful to quickly access exactly the sites one enjoys. However, there are other potential sources to improve the user experience—like showing the user’s bookmarks, too:

![Brave home screen with address bar, tracking data, frequently visited sites—and ample unused space.](https://user-images.githubusercontent.com/1700772/198897093-19b80da7-1d50-41d7-9ba9-271e40cfebbf.png)

## Steps to reproduce

1. Have a home screen with space
2. Add a bookmark

## Actual result

3. Wonder why there’s no quick home screen access to your bookmarks

## Expected result

3. See your bookmarks on the home screen

## Issue reproduces how often

Permanent (not an issue, but a missed feature)

## Version/Channel Information:

n/a

## Device details
   - Install type (ARM, x86):
   - Device type (Phone, Tablet, Phablet): Phone (Pixel 6a)
   - Android version: 13
   
## Brave version

1.45.113

### Website problems only

n/a

### Additional information

n/a

**Comment by kyoyacchi** [comment]
May I ask how did you make them horizontal? I'd like to do it too .

**Comment by j9t** [comment]
> May I ask how did you make them(shortcuts) horizontal? I'd like to do it too .

These are just frequently visited sites, I believe reflecting Brave’s default behavior.
